triton-server/src/notification.ts

```diff
@@ -293,12 +293,7 @@ export const sendValidationEmail = async (
         for (const dataset of validatedDatasets) {
             if (dataset.projectAndRunInfo.validation_status > 0) {
                 const subject = `A Run has been validated.`
-                await sendEmail(
-                    "",
-                    "sequencing-runs@computationalgenomics.ca",
-                    subject,
-                    body,
-                )
+                await sendEmail("", config.email_testing.to, subject, body)
```

Comment:
defaulting to email_testing defeats the point of having the environment variable.

Reply:
we should stick to just one email for validation.

Reply:
clarification is appreciated, be sure to be as clear as possible when writing comments